spec/papers_spec.rb

```diff
@@ -121,6 +121,27 @@
     ])
   end
 
+  it 'is OK with whitelisting specific gems', focus: true do
```

Comment:
@jasonrclark  oops, can you remove the `focus` here? Looks like I forgot to do that.